app/plugins/batch_mode/batch_mode.h

```diff
@@ -62,7 +62,9 @@ class BatchMode : public BatchModeTags
 
 	vkb::FlagCommand categories_flag{vkb::FlagType::ManyValues, "category", "C", "Filter samples by categories"};
 
-	vkb::SubCommand batch_cmd{"batch", "Enable batch mode", {&duration_flag, &wrap_flag, &tags_flag, &categories_flag}};
+	vkb::FlagCommand skip_flag{vkb::FlagType::ManyValues, "skip", "", "Skip a sample by id"};
```

Comment:
How is this intended to work?
ManyValues implies that you can give a list of samples, the text says "Skip a sample by id" which implies a single sample.

If it's supposed to be many, how do I specify the list on the command line? Nothing I've tried has worked.

Reply:
I was patterning it on the `--tags` parameter, which I assumed was functional and could accept multiple values, but that appears to have been incorrect.  It looks like there's a bug in the CLI11 parsing wrapper [here](https://github.com/KhronosGroup/Vulkan-Samples/blob/main/framework/platform/parsers/CLI11.cpp#L108) where the code doesn't do anything different for `OneValue` vs `ManyValues`.  I will add a commit to fix this.

Reply:
This does work now, providing you 'just know' that you have to give multiple "--skip" arguments. e.g "--skip a --skip b". I wonder if there's a way we can make that more obvious in the help text.

Reply:
This flag is aimed primarily at example developers who want to test out that their changes don't break any of the samples (but might need to deal with a known broken sample), so I don't think it needs to be excessively user-friendly to people who are using the examples to learn techniques and therefore are less likely to run in batch-mode in the first place.

Reply:
Agreed on this last point. I explicitly use batch mode to test for regressions! The help message here also makes sense to me in context of the rest of this plugins options